sceptre/plan/actions.py

```diff
@@ -139,6 +139,12 @@ def update(self):
                     {"Key": str(k), "Value": str(v)} for k, v in self.stack.tags.items()
                 ],
             }
+
+            if self.stack.disable_rollback:
```

Comment:
there's code in the above `create` method that says.. [can specify either DisableRollback or OnFailure , but not both](https://github.com/Sceptre/sceptre/blob/master/sceptre/plan/actions.py#L86).  Wouldn't this need to be aware of `OnFailure` as well?

Reply:
I think we should leave it as-is for now. Here's the problem:
In the case of [`create_stack`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudformation/client/create_stack.html), the Boto3 docs say:
> DisableRollback (boolean) –
> 
> Set to true to disable rollback of the stack if stack creation failed. You can specify either DisableRollback or OnFailure, but not both.
>
> Default: false

In the case of [`update_stack`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudformation/client/update_stack.html) however, there is no option for `OnFailure` and the docs simply say:

> DisableRollback (boolean) –
>
> Preserve the state of previously provisioned resources when an operation fails.
>
> Default: False

Ideally, I think the original feature that added `DisableRollback` to Sceptre should have implemented the same logic as specified in Boto3, however we did not. Instead we have simply given `DisableRollback` precedence over `OnFailure`. 

Now it gets better! Because although the code indicates in some places that we have support for `on_failure`, it has never been connected to the CLI! This situation appears to go back to 2018.

Thus there is no actual option for anyone to do something silly like

```
sceptre create --disable-rollback=true --on-failure=do-nothing
```

Because we have never actually implemented `on-failure` so far as I can tell. This deserves to be tracked as a separate tech debt ticket.

Reply:
yikes! although i guess nobody actually cared to complain about `on-failure`